docs/references/connection_pool.rst

```diff
@@ -44,34 +43,36 @@ If the request reaches the timeout, it will be aborted with the following respon
    "hint":null,
    "message":"Timed out acquiring connection from connection pool."}
 
-Getting this error message is an indicator of a performance issue. To solve it, you can:
+.. important::
+
+  Getting this error message is an indicator of a performance issue. To solve it, you can:
 
-- Reduce your queries execution time.
+  - Reduce your queries execution time.
 
-  - Check the request :ref:`explain_plan` to tune your query, this usually means adding indexes.
+    - Check the request :ref:`explain_plan` to tune your query, this usually means adding indexes.
 
-- Reduce the amount of requests.
+  - Reduce the amount of requests.
 
-  - Reduce write requests. Do :ref:`bulk_insert` (or :ref:`upsert`) instead of inserting rows one by one.
-  - Reduce read requests. Use :ref:`resource_embedding`. Combine unrelated data into a single request using custom database views or functions.
-  - Use :ref:`s_procs` for combining read and write logic into a single request.
+    - Reduce write requests. Do :ref:`bulk_insert` (or :ref:`upsert`) instead of inserting rows one by one.
+    - Reduce read requests. Use :ref:`resource_embedding`. Combine unrelated data into a single request using custom database views or functions.
+    - Use :ref:`s_procs` for combining read and write logic into a single request.
 
-- Increase the :ref:`pool growth limit <pool_growth_limit>`.
+  - Increase the :ref:`db-pool` size.
 
-  - Not a panacea since connections can't grow infinitely. Try the previous recommendations before this.
+    - Not a panacea since connections can't grow infinitely. Try the previous recommendations before this.
 
 .. _automatic_recovery:
 
 Automatic Recovery
 ------------------
 
-If the pool loses the connection to the database, it will retry reconnecting using exponential backoff. With 32 seconds being the maximum backoff time between retries.
-
-The retries happen immediately after a connection loss, if :ref:`db-channel-enabled` is set to true(the default). Otherwise they'll happen once a request arrives.
-
-The server reloads the :ref:`schema_cache` when recovering.
+The server will retry reconnecting to the database if connection loss happens.
 
-To notify the client of the next retry, the server sends a ``503 Service Unavailable`` status with the ``Retry-After: x`` header. Where ``x`` is the number of seconds programmed for the next retry.
+- It will retry forever with exponential backoff. 32 seconds being the maximum backoff time between retries. Each of these attempts are :ref:`logged <pgrst_logging>`.
```

Comment:
```suggestion
- It will retry forever with exponential backoff, with a maximum backoff time of 32 seconds between retries. Each of these attempts are :ref:`logged <pgrst_logging>`.
```

I think this is more direct.

Reply:
The hemingway app tells me this form is harder to read

![Screenshot from 2023-06-07 19-23-06](https://github.com/PostgREST/postgrest-docs/assets/1829294/18997b7a-10ba-449e-aaeb-47a3d290874d)

Reply:
Ah, I see. The problem is that a cardinal number shouldn't start a sentence, `32` should be `Thirty-two seconds...` in this case, I think.

Reply:
Ah ok. I will go with your suggestion then. Yellow is kinda fine in hemingwayapp.